conf/messages.en

```diff
@@ -571,14 +571,16 @@ organisation.checkAnswers.title = Check your answers
 
 contactDetails.member.firstName = First name
 contactDetails.member.lastName = Last name
+contactDetails.member.name.heading = Who is the main contact for the organisation?
```

Comment:
Why has this new messages been added? Looking at the ticket, I didn't expect there to be a change to the main heading.